deploy/assisted-service.yaml

```diff
@@ -26,6 +26,18 @@ spec:
           imagePullPolicy: Always
           ports:
             - containerPort: 8090
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 8090
+            initialDelaySeconds: 3
+            periodSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: 8090
+            initialDelaySeconds: 3
```

Comment:
Can you remove the  initialDelaySeconds & periodSeconds. lets use the k8s default values